README.md

```diff
@@ -120,6 +126,22 @@ cli:
     #    cli-flatten-payload-max-poly-as-resource-prop-count: 8
     #    # max properties allowed from flatten of sub-class as param
     #    cli-flatten-payload-max-poly-as-param-prop-count: 8
+
+    expand-operation:

```

Comment:
>expand [](start = 4, length = 6)

"expand" may be confusing because it can mean different thing like expand env var in a string. how about split?

Reply:
Change to split. Thanks